.github/actions/download-rpm/action.yaml

```diff
@@ -3,9 +3,16 @@ description: 'Will download the RPM as prep for container building'
 runs:
   using: "composite"
   steps:
-    - name: Download vertica RPM package
+    - name: Download vertica RPM package to docker-vertica
       shell: bash
       env:
           VERTICA_CE_URL: "https://vertica-community-edition-for-testing.s3.amazonaws.com/XCz9cp7m/vertica-12.0.3-0.x86_64.RHEL6.rpm"
       run: |
         curl $VERTICA_CE_URL -o docker-vertica/packages/vertica-x86_64.RHEL6.latest.rpm
+
+    - name: Download vertica RPM package to docker-vertica-2
```

Comment:
This does two downloads each time it is called. Let's just have the caller specify the target directory.